gateway/src/main/java/de/tum/in/www1/artemis/security/jwt/JWTFilter.java

```diff
@@ -0,0 +1,44 @@
+package de.tum.in.www1.artemis.security.jwt;
```

Comment:
Similar to above, maybe it is not as relevant but it is still in the security directory.
Here is the diff https://www.diffchecker.com/SpzPCbmj
It differs strongly.